Retail spell updates

domzae opened the pull request:
Added some of the recently unlocked soulbind traits
Added ~25 new spells (including #223, #202, #197/#127, and most from #168)
Update/removed some old/wrong/redundant spells
Fixed the whitespace introduced in 9.8 which caused build to error (I think!)

Edit:
New category: Speed Boosts (**I have no idea if this causes any issues in classic, as it will have no entries**)

Kennah229 commented:
(Removed spell, causing blank space in defensive buffs) - [204150] = { type = BUFF_DEFENSIVE }, -- Aegis of Light

I also added this to Blessing of Sacrifice - [199450] = { type = BUFF_DEFENSIVE, parent = 6940 }, -- Blessing of Sacrifice (Ultimate Sacrifice Damage)
It's the dot damage the paladin takes while using ultimate sacrifice on a target. Useful for seeing the paladin that sacrificed the target.

Perhaps change Cyclone to Immunity? [33786] = { type = IMMUNITY }, -- Cyclone

Some other spells that may be useful:
[87023] = { type = BUFF_DEFENSIVE }, -- Cauterize
[25771] = { type = BUFF_OTHER }, -- Forbearance
[132158] = { type = BUFF_OFFENSIVE }, -- Nature's Swiftness
[310143] = { type = BUFF_OTHER }, -- Soulshape
[34709] = { type = BUFF_OTHER }, -- Shadow Sight

domzae commented:
Nice find on the Aegis of Light, I spent way too long looking for that culprit and not finding it in the end. :(
I've added the parent ID's to those spells, thanks!

My argument against putting Cyclone (and similar spells like Diamond Ice, Detainment & Banish) under immunities, is that if somebody uses BigDebuffs to only track CC for example, then they wouldn't show up. Perhaps we could add another category (CROWD_CONTROL_IMMUNE?) or maybe better would be to give those spells crowd_control.priority =+ 1, unless overwritten by a custom priority?

Added those last spells!

Kennah229 commented:
Yeah I literally went through the defensive buffs one by one until I found the broken spell lmao. I noticed there were 6-7 blank spaces a few months ago and now there was just one so I figured you couldn't find it.

You're probably right about Cyclone. I suppose "buff" immunities and "debuff/cc" immunities are somewhat different. May need a separate category for them at some point but for now I suppose people can just raise the custom priority.

Nice call with the speed boosts category.
